docs/guides/code_fsm_app_skill.md

```diff
@@ -0,0 +1,39 @@
+
+Recall that the [{{fsm_app}} skill](../key_concepts/fsm_app_introduction.md) is the core part of a service agent, and it encodes the business logic of the service. Developing the {{fsm_app}} is possibly most demanding step in the development process.
```

Comment:
a service agent -> an agent service

Reply:
I disagree. The subjet here is "agent", not "service". 
(a service agent = an agent of the service)

Reply:
I still think we should avoid using "service agent", it is confusing IMO.

Let me suggest: "Recall that the [{{fsm_app}} skill](../key_concepts/fsm_app_introduction.md) is the core part of the agent that encodes the business logic of the service." or similar

Reply:
fixed